code/game/objects/items/apc_frame.dm

```diff
@@ -46,10 +46,17 @@
 
 /obj/item/wallframe/attackby(obj/item/weapon/W, mob/user, params)
 	..()
+	if(istype(W, /obj/item/weapon/screwdriver))
+		// For camera-building borgs
+		var/turf/T = get_step(get_turf(user), user.dir)

```

Comment:
Are you trying to find the turf the Wallframe *looks* to be on? (eg, the wall)
if so, ```get_turf_pixel()``` will do that.